### PR TITLE
spellcheck

### PR DESCRIPTION
**Pull request by LPardue**

### CODE REVIEW

File: draft-ietf-quic-qlog-main-schema.md [hunk 1]
```diff
@@ -1143,7 +1143,7 @@ non-truncated lengths.
 Note:
 
 : We do not specify explicit header_length or trailer_length fields. In
-most protocols, header_length can be calculated by subtracing the payload_length
+most protocols, header_length can be calculated by sub-tracing the payload_length
```
Comment (lnicco) [inline draft-ietf-quic-qlog-main-schema.md:1146]: subtracting

File: draft-ietf-quic-qlog-main-schema.md [hunk 2]
```diff
@@ -105,7 +105,7 @@ definitions).
 
 The goal of this high-level schema is to provide amenities and default
 characteristics that each logging file should contain (or should be able to
-contain), such that generic and reusable toolsets can be created that can deal
+contain), such that generic and reusable tool sets can be created that can deal
```
Comment (lnicco) [inline draft-ietf-quic-qlog-main-schema.md:108]: this kinda looks fine as it is? but I'll leave it to you, I don't feel strongly

File: draft-ietf-quic-qlog-quic-events.md [hunk 3]
```diff
@@ -1383,7 +1383,7 @@ when data was retransmitted).
 
 Definition:
 
-~~~ ccdl
+~~~ cddl
```
Comment (lnicco) [inline draft-ietf-quic-qlog-quic-events.md:1386]: great catch!

**Review by lnicco — APPROVED** [review]
found one thing to fix. approving to unblock you though. 
Feel free to merge after fixing the typo

File: draft-ietf-quic-qlog-quic-events.md (hunk 3 above)
Comment (marten-seemann) [reply draft-ietf-quic-qlog-quic-events.md:1386]: oops

**Review by marten-seemann — APPROVED** [review]
lgtm, modulo @lnicco's comments

File: draft-ietf-quic-qlog-main-schema.md (hunk 2 above)
Comment (LPardue) [reply draft-ietf-quic-qlog-main-schema.md:108]: This was my opinionated spellchecker, I'll revert.

**Pull request merged**